Lombiq.Privacy.Samples/Recipes/Lombiq.Privacy.Samples.Content.recipe.json

```diff
@@ -0,0 +1,504 @@
+                },
+                "ActivityMetadata": {
+                  "Title": "Create competitor item"
+                }
+              }
+            },
+            {
+              "ActivityId": "46bbnrx4k6wbw197vpmdz212dz",
+              "Name": "HttpRedirectTask",
+              "X": 1050,
+              "Y": 160,
+              "IsStart": false,
+              "Properties": {
+                "ActivityMetadata": {
+                  "Title": "Redirect to details"
+                },
+                "Location": {
+                  "Expression": "{{ Workflow.Properties[\"ContentItem\"] | display_url }}"
+                },
+                "Permanent": false
+              }
+            },
+            {
+              "ActivityId": "4n7zznh3ssm9078wyva0rq45d2",
+              "Name": "HttpRedirectTask",
+              "X": 1120,
+              "Y": 30,
+              "IsStart": false,
+              "Properties": {
+                "ActivityMetadata": {
+                  "Title": null
+                },
+                "Location": {
+                  "Expression": "/error"
+                },
+                "Permanent": false
+              }
+            }
+          ],
+          "Transitions": [
+            {
+              "Id": 0,
+              "SourceActivityId": "45g5pkc6tpy86v71mdxq4685d8",
+              "SourceOutcomeName": "Done",
+              "DestinationActivityId": "4bnngn8hxa684201xsaq9fm1wh"
+            },
+            {
+              "Id": 0,
+              "SourceActivityId": "4bnngn8hxa684201xsaq9fm1wh",
+              "SourceOutcomeName": "Valid",
+              "DestinationActivityId": "474fg602kmn9z0xsn389w51mnf"
+            },
+            {
+              "Id": 0,
+              "SourceActivityId": "474fg602kmn9z0xsn389w51mnf",
+              "SourceOutcomeName": "Done",
+              "DestinationActivityId": "46bbnrx4k6wbw197vpmdz212dz"
+            },
+            {
+              "Id": 0,
+              "SourceActivityId": "4bnngn8hxa684201xsaq9fm1wh",
+              "SourceOutcomeName": "Invalid",
+              "DestinationActivityId": "4n7zznh3ssm9078wyva0rq45d2"
+            },
+            {
+              "Id": 0,
+              "SourceActivityId": "474fg602kmn9z0xsn389w51mnf",
+              "SourceOutcomeName": "Failed",
+              "DestinationActivityId": "4n7zznh3ssm9078wyva0rq45d2"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "content",
+      "data": [
+        {
+          "ContentItemId": "4ftg1gchvpstvt5zz4hhnr84w0",
+          "ContentItemVersionId": "491rv1zfbdvbxxzcbvgbks4mn4",
```

Comment:
Please give these both Page content items human readable content item ID, and `[js:uuid()]` version ID.